src/gql/queries/get-code-changes.ts

```diff
@@ -13,6 +13,7 @@ export const GET_CODE_CHANGES = gql`
           additions
           deletions
           diffLink
+          description
```

Comment:
was this intentional?

Reply:
I was getting a typescript error for mismatched types this was the quickest fix at the time. I think this will be fixed in your pr.  https://github.com/evergreen-ci/spruce/pull/564/files